mlxtend/classifier/stacking_cv_classification.py

```diff
@@ -0,0 +1,270 @@
+# Stacking CV classifier
+
+# Sebastian Raschka 2014-2016
+# mlxtend Machine Learning Library Extensions
+#
+# An ensemble-learning meta-classifier for stacking
+# Authors: Reiichiro Nakano <github.com/reiinakano>
+#          Sebastian Raschka <sebastianraschka.com>
+#
+# License: BSD 3 clause
+
+from sklearn.base import BaseEstimator
+from sklearn.base import ClassifierMixin
+from sklearn.base import TransformerMixin
+from sklearn.base import clone
+from sklearn.utils.validation import check_is_fitted
+from sklearn.pipeline import _name_estimators
+from sklearn.externals import six
+import numpy as np
+from sklearn.cross_validation import StratifiedKFold
+from sklearn.cross_validation import KFold
+
+
+class StackingCVClassifier(BaseEstimator, ClassifierMixin, TransformerMixin):
+
+    """A Stacking CV classifier for scikit-learn estimators for classification.
+
+    Parameters
+    ----------
+    classifiers : array-like, shape = [n_regressors]
```

Comment:
I think this should say `shape = [n_classifiers]` not `shape = [n_regressors]` :)